packages/core/test/model/chain/oasis.test.ts

```diff
@@ -1,33 +1,43 @@
 import { expect } from 'chai'
 import { TEST_ADDRESS, TEST_TX } from './defaults'
-import { OasisEmerald } from '../../../src'
+import { OasisEmerald, OasisEmeraldTestnet } from '../../../src'
 
 describe('Oasis Emerald Chain', () => {
   it('getChainId', () => {
     expect(OasisEmerald.chainId).to.equal(42262)
+    expect(OasisEmeraldTestnet.chainId).to.equal(42261)
   })
 
   it('getChainName', () => {
     expect(OasisEmerald.chainName).to.eq('OasisEmerald')
+    expect(OasisEmeraldTestnet.chainId).to.equal('OasisEmeraldTestnet')
```

Comment:
change to chainName like higher in the main OasisEmerald test.

Reply:
Good catch